client/web/divkit/src/devCustomComponents.ts

```diff
@@ -53,7 +53,9 @@ class NewCustomCard extends HTMLElement {
             const minutes = Math.floor(this._counter / 60);
             const seconds = this._counter % 60;
             this._text.textContent = String(minutes).padStart(2, '0') + ':' + String(seconds).padStart(2, '0');
-
+            if (this._counter > 1) {
+                return;
+            }
```

Comment:
Added timer stop after 2 seconds of ticking to create proper screenshots in interactive tests of customcard (without it test may be flaky, because time spend on screenshot differs from time to time)